src/components/ErrorPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { WarningAlt } from "@carbon/icons-react";
+
+/**
+ * Under Construction Page
+ */
+const ErrorPage = () => {
+  return (
+    <div className="underConstruction flexDirectionColumn">
+      <WarningAlt size="128" />
+      <h4>Error 404: Page Not Found</h4>
```

Comment:
let's mimic IBM Cloud a bit here. 

change to "Unfortunately, we couldn’t find the page you were looking for, but here are some helpful places to start from:" then list links for Projects, About, and Tutorial. 

<img width="200" alt="Screenshot 2023-10-19 at 3 52 08 PM" src="https://github.com/IBM/icse-react-assets/assets/11401353/05a2fcd0-3860-41ef-98e7-75d5bc5fbf0a">